### PR TITLE
IAM Credentials for S3 bucket 

### PR DESCRIPTION
**Pull request by pbench**
Try to adapt what is done in https://github.com/hove-io/asgard/pull/216/files

### CODE REVIEW

File: source/tyr/tyr/minio.py [hunk 1]
```diff
@@ -1,42 +0,0 @@
-# coding: utf-8
```
Comment (woshilapin) [inline source/tyr/tyr/minio.py:1]: This file was not used at all? :open_mouth:

**Review by woshilapin — APPROVED** [review]
We assume this new version of `tyr` is never going to be deployed on-premise? Note that I see the default value to AWS a bit problematic as an open-source repository: maybe we can default to `None` instead?

File: source/tyr/tyr/minio.py (hunk 1 above)
Comment (pbench) [reply source/tyr/tyr/minio.py:1]: well it was used only for reading some config variables, which are used only in one place. So i figured it was really not worth it to have a file for that

**Comment by pbench** [comment]
Work continued in https://github.com/hove-io/navitia/pull/3849

**Pull request closed without merging**